src/clarity-angular/data/datagrid/render/header-renderer.spec.ts

```diff
@@ -27,8 +27,8 @@ export default function(): void {
             context = this.create(
                 DatagridHeaderRenderer, SimpleTest,
                 [MOCK_ORGANIZER_PROVIDER, MOCK_DOM_ADAPTER_PROVIDER, Sort, FiltersProvider, Page, StateDebouncer]);
-            domAdapter = context.getClarityProvider(DomAdapter);
-            organizer = context.getClarityProvider(DatagridRenderOrganizer);
+            domAdapter = <MockDomAdapter>context.getClarityProvider(DomAdapter);
```

Comment:
Are these type assertion changes in Datagrid related to Tabs?

Reply:
As I mentioned in the PR's description, a small change in the test helpers was needed for me to be pass typescript compilation on the tabs tests. And that change had a little impact on several Datagrid unit tests.